implementations/rust/ockam/ockam_command/src/authority/static/create/long_about.txt

```diff
@@ -0,0 +1 @@
+This command will a new pre-configured authority node with the needed services to issue and verify cryptographically signed credentials.
```

Comment:
`will` -> `will create` / `will start`.

Not sure if pre-configured add much,   I'd  let it as 
`This command starts an authority node with the needed services to issue and verify cryptographically signed credentials.`

Reply:
fixed, thanks!